roles/ora-host/tasks/main.yml

```diff
@@ -178,6 +209,31 @@
     - "{{ sysctl_entries }}"
   tags: sysctl
 
+- name: Get the ID of the oinstall group
+  shell:
+    cmd: "getent group oinstall | cut -d: -f3"
+  register: oinstall_group_id
+  changed_when: false
+  when:
+    - ansible_distribution == 'RedHat'
+    - ansible_distribution_major_version == '9'
+
+
+# Setting vm.hugetlb_shm_group to the ID of the "oinstall" group to allow its members to create 
+# SysV shared memory segment using hugetlb page. 
+# By default only the root user has permissions to create shared memory segments.
+# This is needed to prevent the occurrence of "ORA-27125: unable to create shared memory segment" error.
+- name: Set hugetlb_shm_group to the ID of the oinstall group
+  sysctl:
+    name: vm.hugetlb_shm_group
+    value: "{{ oinstall_group_id.stdout }}"
+    state: present
+    reload: true
+  become: true
+  when:
+    - ansible_distribution == 'RedHat'
+    - ansible_distribution_major_version == '9'
```

Comment:
Is this actually RHEL9 specific?  https://access.redhat.com/solutions/1610713 lists is as being hte case with RHEL6+. 
And, for that matter, shouldn't we expect the same in Oracle Linux too?

Reply:
The ORA-27125 error seems to only impact RH9. When I used the bms-toolkit, cloned from the toolkit-ol9 branch, on RH8, I didn't encounter this error, and the installation finished successfully. See the installation log here: https://gist.github.com/alex-basinov/f8c5ee903f24b775cc988ef6ba0f0fd8
It also didn't fail on OL9: https://gist.github.com/alex-basinov/bda41a2af7f2b12fb48fe2320fc36809

Note that I haven't tested on RH6 and RH7. However, considering that this error has not appeared previously, it's reasonable to conclude that RH9 is the only version impacted.

Reply:
Fair enough;  we can modify later if we get reports of issues.